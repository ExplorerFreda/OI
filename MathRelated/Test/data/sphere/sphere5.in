7
17310.500000 9308.000000 16136.000000 7421.500000 13528.000000 17407.000000 5332.000000
107.000000 5943.000000 2515.000000 7770.500000 19673.500000 1708.000000 13854.500000
2670.500000 13632.000000 3070.500000 1998.000000 19117.500000 11815.500000 15857.000000
3125.500000 11171.000000 4057.000000 17865.500000 13127.500000 13764.500000 1670.500000
10888.500000 13482.000000 11200.500000 5807.500000 6372.500000 17580.000000 9202.000000
18119.000000 16432.500000 9014.500000 15433.000000 17443.000000 14271.500000 15547.000000
16834.500000 6993.000000 585.500000 19689.500000 14167.500000 10163.500000 8265.500000
7576.500000 19043.000000 13861.000000 5458.000000 445.000000 16465.000000 6017.000000
