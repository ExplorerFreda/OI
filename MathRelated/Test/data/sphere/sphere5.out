17169.936 26107.242 4704.647 2240.453 16008.218 -7634.507 -4437.186
